### PR TITLE
Add various coins

### PR DESCRIPTION
**Pull request by erasmospunk**
Coins added:
Posw coin
Shreeji
Bitcoinplus
Internet of People
Nexus
InsaneCoin
BritCoin
Compcoin
Crown

### CODE REVIEW

File: slip-0044.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ index | hexa       | coin
 44    | 0x8000002c | [Particl](https://particl.io/)
 45    | 0x8000002d | [Argentum](http://www.argentum.io)
 46    | 0x8000002e | [Libertas](https://github.com/dangershony/Libertas)
-47    | 0x8000002f |
-48    | 0x80000030 |
+47    | 0x8000002f | [Posw coin](https://poswallet.com/maint.php)
```
Comment (prusnak) [inline slip-0044.md]: I don't think you want to use /maint.php here

File: slip-0044.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ index | hexa       | coin
...
+48    | 0x80000030 | [Shreeji](https://github.com/SMJBIT/SHREEJI.git)
```
Comment (prusnak) [inline slip-0044.md]: Don't use .git at the end

File: slip-0044.md [hunk 3]
```diff
@@ -93,14 +93,14 @@ index | hexa       | coin
...
+65    | 0x80000041 | [Bitcoinplus](https://bitcoinplus.org)
+66    | 0x80000042 | [Internet of People](http://www.fermat.org)
+67    | 0x80000043 | [Nexus](http://www.nexusearth.com/)
+68    | 0x80000044 | [InsaneCoin](http://insanecoin.com)
 69    | 0x80000045 | [OKCash](https://github.com/okcashpro/)
-70    | 0x80000046 |
-71    | 0x80000047 |
-72    | 0x80000048 |
+70    | 0x80000046 | [BritCoin](https://britcoin.com)
+71    | 0x80000047 | [Compcoin](www.compcoin.com)
```
Comment (prusnak) [inline slip-0044.md]: Use https://compcoin.com/ here

**Comment by erasmospunk** [comment]
Fixed

**Comment by prusnak** [comment]
Thx!

**Pull request merged**